### PR TITLE
Tree fizz buzz

### PR DESCRIPTION
**Pull request by c0d3cisco**
- [x] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
       - [x] Summary, Description, Approach & Efficiency, Solution
       - [x] Picture of whiteboard
       - [x] Link to code
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
       - [x] “Happy Path” - Expected outcome
       - [x] Expected failure
       - [x] Edge Case (if applicable/obvious)

### CODE REVIEW

File: javascript/trees/tree-fizz-buzz/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const { KaryTree, KaryNode } = require('../index.js');
+const { Queue } = require('../../stack_Queue/index.js');
```
Comment (lizkavalski) [inline javascript/trees/tree-fizz-buzz/index.js:4]: check your pathing
